src/components/RangeInput.vue

```diff
@@ -11,7 +11,7 @@
     >
       <form
         :class="suit('form')"
-        @submit.prevent="refine({ min: minInput, max: maxInput })"
+        @submit.prevent="refine({ min: minInput || values.min, max: maxInput || values.max })"
```

Comment:
I've just noticed that this is going to break when `minInput` or `maxInput` are set to `0`. Since `0` is falsy in JS, it'll end up taking the right value of the `||`

Reply:
Thanks @dotboris for the review and finding it out !
https://github.com/algolia/vue-instantsearch/pull/777/commits/5189f33bd60d6279d08a194074423ddd891df8b7